radio/wi-fi/wps.md

```diff
@@ -1,10 +1,132 @@
-# 🛠️ WPS
+# WPS
 
-## Recon
+## Theory
+
+Wi-Fi Protected Setup (WPS) is a simplified configuration protocol designed to make connecting devices to a secure wireless network easier.
+WPS employs an 8-digit PIN for user network connection, verifying the first 4 digits before proceeding to check the remaining 4. This allows for a potential Brute-Force attack on the first set of digits followed by the second set, with a total of only 11,000 possible combinations.
+
+There are three main methods for setting up a connection using WPS :
+
+* **Push Button Configuration (PBC) :** Press the WPS button on the router and then activate the WPS function on the device to connect. The two devices will automatically establish a connection.
+
+* **PIN-based Configuration :** Each WPS-enabled device has a unique PIN code. It can be entered on the router through its web interface, or vice versa, to establish the connection.
+
+* **Near-field communication Configuration :** In which the user has to bring the new client close to the access point to allow a near field communication between the devices. NFC Forum–compliant RFID tags can also be used.
```

Comment:
`near field communication` --> `Near Field Communication`, so that the "NFC" is more obvious 😉

Reply:
ok